demos/gallery/samples/layout/Grid.js

```diff
@@ -0,0 +1,21 @@
+import { Bar, Column, Step } from "@hpcc-js/chart";
+import { Grid } from "@hpcc-js/layout";
+
+const columns = ["Category", "Value-1", "Value-2"];
+const data = [
+    ["A", 34, 21],
+    ["B", 55, 34],
+    ["C", 54, 90],
+    ["D", 80, 153],
+    ["E", 86, 92],
+    ["F", 144, 233]
+];
+
+const grid = new Grid()
+    .target("target")
+    .setContent(0, 0, new Bar().columns(columns).data(data))
+    .setContent(1, 0, new Column().columns(columns).data(data))
+    .setContent(0, 1, new Step().columns(columns).data(data))
+    ;
+
+grid.render();
```

Comment:
Why does the following not work:
```
const grid = new Grid()
    .target("target")
    .setContent(0, 0, new Bar().columns(columns).data(data))
    .setContent(1, 0, new Column().columns(columns).data(data))
    .setContent(0, 1, new Step().columns(columns).data(data))
    .render()
    ;
```

Reply:
It does work initially on page load, but then `playground.ts -> syncJavaScript` removes the lines between `.target` and `.render` if they contain a method call that isn't a publish parameter.

Reply:
You need to move the setContent to above the `target` call as they are not PP and that should fix also.